pyseext/grid_helper.py

```diff
@@ -399,6 +403,32 @@ def get_row(self, grid_cq: str, row_data: Union[int, dict], should_throw_excepti
 
         raise GridHelper.RowNotFoundException(grid_cq, row_data)
 
+    def get_row_store(self, grid_cq: str, row_data: Union[int, dict], should_throw_exception: bool = True) -> dict:
```

Comment:
As discussed in the JS, this gets the data for a row, not a store.
Make sure update the template and comments as needed too please.

Reply:
all updated